pkg/apis/numaflow/v1alpha1/vertex_types.go

```diff
@@ -253,6 +253,18 @@ func (v Vertex) GetPodSpec(req GetVertexPodSpecReq) (*corev1.PodSpec, error) {
 		return nil, err
 	}
 
+	var readyzInitDeploy, readyzPeriodSeconds, readyzTimeoutSeconds, readyzFailureThreshold int32 = 5, 10, 30, 6 // default values
```

Comment:
Should we store these default numeric values as constants, i see them getting reused in multiple places

Reply:
done.